Packs/Jira/Integrations/JiraV3/JiraV3_description.md

```diff
@@ -74,8 +72,6 @@ The integration uses the *offline_access* scope, in order to retrieve refresh to
 3. To create a new `Application link`, which will be used to integrate Cortex XSOAR with Jira:
     a. Click **Application links** under `Integrations`, found on the left side menu.
     b. Click **Create link** and choose **External application** with the **Incoming** direction.
-4. Fill in the required details as explained in the page and choose the permission **Write**.
-5. Once the link is created, you will be able to see `Client ID`, and the `Client secret`, which are required in the configuration screen. Copy these values and paste them into the respective fields in the configuration screen.
```

Comment:
Since we are missing the reference of the **Callback URL**, we can change point 4 to be:

Fill in the required details as explained in the page, by inserting a **Callback URL**, and choosing the permission **Write**.